api/src/pcapi/core/offers/api.py

```diff
@@ -136,6 +137,7 @@ def create_offer(
     visual_disability_compliant: bool,
     booking_contact: str | None = None,
     booking_email: str | None = None,
+    date_published: datetime.datetime | None = None,
```

Comment:
[nommage] `date_published` étant au passé, on pourrait penser qu'il s'agit de la date à laquelle l'offre "à été publiée", or on est plutôt dans l'anticipation du coup je mettrais plutôt une truc comme `publication_date`

Reply:
Pour le nommage je suis parti du nom des colonnes dans le modèle `Offer`, ça me semblait cohérent de nommer la colonne `datePublished` comme il y a déjà les colonnes `dateCreated`, `dateModifiedAtLastProvider` et `dateUpdated`.
Pour le temps futur ou passé j'ai l'impression que `published` convient dans les 2 cas :
 - will be published
 - was published

Reply:
(du coup dans le serializer j'ai donné le même nom (en snake_case ou camelCase selon la convention du serializer) que le nom de la colonne)

Reply:
@ogeber Renommage `datePublished` -> `publicationDate` effectué